### DISCUSSION TITLE
Happy new year !!🎊

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/grandorgue#929] [General]
Happy New Year 2022 everyone!

Many big thanks to all the developers who allow us to play our favorite instruments. 
Thank you for all the work that has been understood so far, and thank you in advance for all the work that will be accomplished during this year 2022.

May this new year sees exponential development of Grand Orgue.

Happy New Year, peace, joy, health for you and your loved ones!

**Comment by oleg68** [comment]
Happy new year!

**Reply by rousseldenis** [reply]
Happy new years folks!
